src/Horarium.Mongo/Horarium.Mongo.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <Description>Horarium.Mongo</Description>
+        <Version>1.0.0$(VersionSuffix)</Version>
+        <Title>Horarium.Mongo</Title>
+        <Authors>bobreshovr</Authors>
+        <PackageDescription>Horarium.Mongo is the .Net library for support MongoDB in Horarium</PackageDescription>
```

Comment:
Horarium.Mongo is the .Net library for support MongoDB in Horarium

May be?

> to support
or
> for supporting

Reply:
done